components/headers/H1.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import styled from 'styled-components';
+import { getString } from "../../lib/richText";
+
+const H1Style = styled.h1`
+    font-size: 80px;
+    font-family: cormorant-garamond, serif; 
+    font-style: italic; 
+    font-weight: 900px;
+`;
+
+export default function H1({ h1Data }) {
+
+    return (
+        <>
+            <H1Style>{getString(h1Data)}</H1Style>
```

Comment:
Also: a suggestion. You can utilize children here to have this do what you want in a simpler way.
This could be the goal:
`<H1>{homePageTitle}</H1>`

Then in the component:

```
export default function H1(props) {
  return <H1Style>{getString(props.children)}</H1Style>;
}
```
Both work fine, but up to you. If you have multiple children in the component this falls apart.

Reply:
Thanks for the suggestion! I'll look into this~